cmd/cli/envcli.go

```diff
@@ -5,66 +5,66 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/smartcontractkit/helmenv/environment"
+	"github.com/smartcontractkit/helmenv/tools"
 	"github.com/urfave/cli/v2"
 	"os"
 )
 
 func init() {
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr}).Level(zerolog.InfoLevel)
+}
+
+var presetFlag = &cli.StringFlag{
+	Name:     "preset",
+	Aliases:  []string{"p"},
+	Usage:    "filepath to the environment preset",
+	Required: true,
+}
+
+var environmentFlag = &cli.StringFlag{
+	Name:     "environment",
+	Aliases:  []string{"e"},
+	Usage:    "filepath to the environment file",
+	Required: true,
 }
 
 func main() {
 	app := &cli.App{
-		Name:                 "envcli",
-		Usage:                "setup default environment",
-		EnableBashCompletion: true,
-		Flags: []cli.Flag{
-			&cli.StringFlag{
-				Name:     "kubectlProcess",
-				Aliases:  []string{"kctl"},
-				Value:    "/usr/local/bin/kubectl",
-				Usage:    "kubectl process name to use forking for port forwarding",
-				Required: false,
-			},
-			&cli.StringFlag{
-				Name:     "preset",
-				Aliases:  []string{"p"},
-				Usage:    "environment preset name",
-				Required: true,
-			},
-		},
-		Before: func(c *cli.Context) error {
-			presetPath := c.String("preset")
-			if err := environment.IsCLIAllowed(presetPath); err != nil {
-				return err
-			}
-			return nil
-		},
+		Name:  "envcli",
+		Usage: "setup default environment",
 		Commands: []*cli.Command{
 			{
 				Name:    "new",
 				Aliases: []string{"n"},
 				Usage:   "create new environment from preset file",
+				Flags:   []cli.Flag{presetFlag},
 				Action: func(c *cli.Context) error {
-					presetPath := c.String("preset")
-					_, err := environment.NewEnvironmentFromPreset(presetPath)
+					preset := c.String("preset")
+					e, err := environment.NewEnvironmentFromPresetFile(tools.ChartsRoot, preset)
 					if err != nil {
 						return err
 					}
+					log.Info().
+						Str("environmentFile", fmt.Sprintf("%s.yaml", e.Config.NamespaceName)).
+						Msg("Environment setup and written to file")
 					return nil
 				},
 			},
 			{
 				Name:    "connect",
 				Aliases: []string{"c"},
 				Usage:   "connects to selected environment",
+				Flags:   []cli.Flag{environmentFlag},
 				Action: func(c *cli.Context) error {
-					presetPath := c.String("preset")
-					e, err := environment.LoadEnvironment(presetPath)
+					environmentPath := c.String("environment")
+					e, err := environment.LoadPersistentEnvironment(environmentPath)
 					if err != nil {
 						return err
 					}
-					if err := e.Connect(); err != nil {
+					if !e.Config.PersistentConnection {
```

Comment:
I think it's better to forbid every possible interaction when using CLI (Debugging), so we can have only two modes:
1. CI run with `false/false` 
2. Local run where you setup env yourself and do that only once.
Local run will save some resources when debugging and make you write tests more carefully with, for example, UUIDs and cleanup, so if we decide to "batch" the tests later we don't need to "fix" them again.